sdk/formrecognizer/azure-ai-formrecognizer/samples/async_samples/sample_copy_model_async.py

```diff
@@ -44,15 +44,15 @@ async def copy_model_async(self):
         target_region = os.environ["AZURE_FORM_RECOGNIZER_TARGET_REGION"]
         target_resource_id = os.environ["AZURE_FORM_RECOGNIZER_TARGET_RESOURCE_ID"]
 
-        # [START authorize_copy_target_async]
+        # [START get_copy_authorization_async]
```

Comment:
nit: you might want to mention that this is for a custom model in the description of this sample